content/broker/broker-transition/usage/soap/_index.en.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Usage SOAP
+linktitle: Usage SOAP
+description: Difference in SOAP operation usage between Altinn 2 and Altinn 3 transitioned services.   
+tags: [architecture, solution]
+toc: false
+weight: 1
+---
+
+## Difference in usage between existing Altinn 2 SOAP Operations and Altinn 3 SOAP operations that have been transition to Altinn 3.
+SOAP operations for Altinn 2 Broker Service are spread over 3 endpoints.
+For most requests there will be no functional difference between Altinn 2 and Altinn 3 requests.
+
+|Service|Endpoint|Description|
+|-----------------------|---------------------------------------------------------------------------|-------------------------------|
+|BrokerService          |[Service WS](https://www.altinn.no/ServiceEngineExternal/BrokerServiceExternal.svc)              | Handles metadata requests.    |
+|                       |[Service Basic](https://www.altinn.no/ServiceEngineExternal/BrokerServiceExternalBasic.svc)         |                               |
+|                       |[Service EC](https://www.altinn.no/ServiceEngineExternal/BrokerServiceExternalEC2.svc)            |                               |
+|BrokerService Streamed |[Streamed WS](https://www.altinn.no/ServiceEngineExternal/BrokerServiceExternalStreamed.svc)      | Handles file data requests.   |
+|                       |[Streamed Basic](https://www.altinn.no/ServiceEngineExternal/BrokerServiceExternalStreamedBasic.svc) |                               |
+|                       |[Streamed EC](https://www.altinn.no/ServiceEngineExternal/BrokerServiceExternalStreamedEC2.svc)    |                               |
+|Receipt                |[Receipt WS](https://www.altinn.no/IntermediaryExternal/ReceiptExternal.svc)                     | Handles receipt requests.     |
+|                       |[Receipt Basic](https://www.altinn.no/IntermediaryExternal/ReceiptExternalBasic.svc)                |                               |
+|                       |[Receipt EC](https://www.altinn.no/IntermediaryExternal/ReceiptExternalEC2.svc)                 |                               |
+
+For examples of SOAP implementation see [existing documentation (norwegian only)](https://altinn.github.io/docs/api/soap/grensesnitt/#brokerservice--formidlingstjenester-ws)
+
+### BrokerService Endpoint
+
+The BrokerService endpoint contains functions related to BrokerService metadata requests.
+
+#### InitiateBrokerService
+
+No functional difference for Transitioned services.
+
+#### GetAvailableFiles
+
+No functional difference for Transitioned services. In order to retrieve Altinn 3 files, the service code and the service edition code parameters must be supplied. 
```

Comment:
consider rewording this according to the same principle as for Rest: If the consumer does not specify SC/SEC they need to do this in order to get files for the transitioned service(s).

Reply:
done